check_process

```diff
@@ -22,9 +22,10 @@
 	Level 1=auto
 	Level 2=auto
 	Level 3=auto
-# Level 4: 
-	Level 4=0
-# Level 5: 
+# Level 4: If the app supports LDAP and SSOwat, turn level 4 to '1' and add a link to an issue or a part of your code to show it.
+# If the app does not use LDAP nor SSOwat, and can't use them, turn level 4 to 'na' and explain as well.
+	#no ldap for now
+	Level 4=na
```

Comment:
I really think we should not put `na` here ... My interpretation of this setting is that `na` means it's not relevant for this app to integrate LDAP/SSO (because, for instance, there is no notion of account in the app, or there wouldn't be any interest in integrating with yunohost)

But that's clearly not the case for Wekan since there are account, and there's a signficant interest with integrating those with yunohost's ldap. Not to mention that it is now technically doable. So we should really put '0'.

Yes, that implies that the app is stuck to level 3, but imho that's an issue with the design of levels and we shouldn't fix this by putting '1' or 'na' in apps that could have LDAP/SSO integration and in fact don't ...

(Sorry for the rant, that's not really against you but I have seen this a few times and it is in fact not clear for everybody how to handle this level 4 thing i believe :/ )

Reply:
I agree with you to put Level 4=0, as there is LDAP available for Wekan but not already implemented in wekan_ynh: https://github.com/wekan/wekan/wiki/LDAP